docs/dev/framework/cron.md

```diff
@@ -3,22 +3,19 @@ title: Cron
 description: "Contao's cron functionality."
 ---
 
+Contao periodically executes some tasks via its own cron functionality. These include mainly cleanup tasks such as
 
-Contao periodically executes some tasks via its own cron functionality. The following
-is a list of tasks executed by Contao's own bundles: 
-
-| Task                                   | Interval |
-|----------------------------------------|----------|
-| Generate calendar RSS feeds            | daily    |
-| Purge expired comment subscriptions    | daily    |
-| Purge temp folder                      | daily    |
-| Purge search cache                     | daily    |
-| Generate XML sitemap                   | daily    |
-| Purge expired registrations            | daily    |
-| Purge expired Opt-In tokens            | daily    |
-| Generate news RSS feeds                | daily    |
-| Purge expired newsletter subscriptions | daily    |
+* Purge expired comment subscriptions
+* Purge expired registrations
+* Purge expired Opt-In tokens
+* etc.
 
+Because all cronjobs are registered as services and tagged using the `contao.cronjob` tag, you can find all
```

Comment:
```suggestion
Starting with Contao **5** all cronjobs are registered as services and tagged using the `contao.cronjob` tag. Thus you can find all
```

Reply:
I wouldn't document this, the legacy crons are also all found there :) You'll need to debug which ones exactly but you can. Right now, there's no info how to do this in 4.13.

Reply:
Hm not sure what you mean? In Contao 5 you can find all cronjobs this way. There are no legacy Cronjobs in Contao 5.

Reply:
Yeah but why would you add this info? It's the same in 4.13.

Reply:
They are all tagged as `contao.cronjob`. There is just one `LegacyCron` that delegates to the `$GLOBALS` thing. But it's still better to know that than knowing nothing :D

Reply:
Yes, you can. You `dump()` in `LegacyCron`.

Reply:
I meant you cannot find all cronjobs via `debug:container --tag contao.cronjob`.

Reply:
I keep telling you "yes you can".

Reply:
In Contao 4.13 `debug:container --tag contao.cronjob` will not tell you out of the box which cronjobs are registered, as there can still be legacy cron jobs (and even within Contao there are still some that are registered this way). Someone reading the documentation would assume otherwise, hence my suggestion to change the wording to make sure that everyone understands that this command only reliably tells you the registered cronjobs starting with Contao 5.

Reply:
I disagree. Again, right now, **there is no mention about legacy crons**. Nothing. You have to know about `$GLOBALS['TL_CRON']`. It's not mentioned on this page of the docs at all. So the current status of the docs is: You know **zero** about legacy crons.

With this PR, when running `debug:container --tag contao.cronjob` you will see `LegacyCron` and you will understand that there is a cron and you can see that this one calls `$GLOBALS['TL_CRON']`. 

So in any case this PR is an improvement over the current docs - for all Contao versions.

Reply:
> I disagree. Again, right now, there is no mention about legacy crons. Nothing. You have to know about $GLOBALS['TL_CRON'].

Legacy cron jobs are documented here: https://docs.contao.org/dev/framework/cron/#registering-cron-jobs

> With this PR, when running debug:container --tag contao.cronjob you will see LegacyCron and you will understand that there is a cron and you can see that this one calls $GLOBALS['TL_CRON'].

I don't think people will draw this inference at all. If anything, we need to document how to also view the legacy cron jobs.

Reply:
I give up then, feel free to do whatever with this PR, sorry but I don't have the energy for this now.

Reply:
I will add a note on a way to view the cron jobs in Contao 4.13 on top of this change (e.g. adding `dump($GLOBALS['TL_CRON']` to the `LegacyCron` service like you suggested.